graph/builder.go

```diff
@@ -37,6 +190,114 @@ func (b *Builder) Start() error {
 
 	log.Info("Builder starting")
 
+	bestHash, bestHeight, err := b.cfg.Chain.GetBestBlock()
+	if err != nil {
+		return err
+	}
+
+	// If the graph has never been pruned, or hasn't fully been created yet,
+	// then we don't treat this as an explicit error.
+	if _, _, err := b.cfg.Graph.PruneTip(); err != nil {
```

Comment:
So typically this is something the router does before it starts up to ensure that it has the most up to date view of the graph. To retain the prior behavior, I think we may need to add a synchronous handoff between this and the router. Also as this might affect the graph itself, if we don't do this _before_ the router fully starts up, then its' cached in memory graph will be out of date. 

On the other hand, those running lnd on mobile platforms have requested that they be able to skip this all together to acheive a faster start up time... 

cc @djkazic

Reply:
Yep, resolving here.